src/test/rgw/ratelimit_sim/README.txt

```diff
@@ -0,0 +1,19 @@
+This tool has been built to evaluate the correctness of the rate limiting algorithm
+
+Compile it:
+g++ -pthread -ltbb -lboost_coroutine -lboost_system -lboost_program_options -std=c++2a simulator.cc -o simulator
```

Comment:
this stuff should just be in a CMakeLists.txt

> -std=c++2a

does anything here actually require anything past c++17?